### PR TITLE
M1 commands

### PR DESCRIPTION
**Pull request by jluhrs**

### CODE REVIEW

File: modules/server/src/main/scala/navigate/server/NavigateEngine.scala [hunk 1]
```diff
@@ -456,6 +417,20 @@ object NavigateEngine {
         }
         .map(_._2)
         .flattenOption
+
+    override def m1Park: F[Unit] = ???
```
Comment (cquiroz) [inline modules/server/src/main/scala/navigate/server/NavigateEngine.scala:421]: Should we implement these?

Comment (jluhrs) [reply modules/server/src/main/scala/navigate/server/NavigateEngine.scala:421]: Ups

Comment (jluhrs) [reply modules/server/src/main/scala/navigate/server/NavigateEngine.scala]: OK, fixed

**Review by cquiroz — APPROVED** [review]
LGTM

**Pull request merged**